packages/amplication-client/src/Purchase/FAQ.tsx

```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
+            <div>
+              Amplication can faciliate the modernization of your legacy system.
+              We offer a suite of tools and features designed to streamline this
+              process. Those capabilities include:
+              <br />
+              <ul>
+                <li>
+                  <strong>DB Schema Import</strong>: Simplify the migration of
+                  extensive legacy data into new systems, preserving the
+                  integrity and structure of historical data.
+                </li>
+                <li>
+                  <strong>Data Modeling & Event Brokering</strong>: Enables the
+                  transition from monolithic architectures to microservices,
+                  allowing for the creation of scalable and manageable backends.
+                </li>
+                <li>
+                  <strong>Customizable Code Generation</strong>: Ensures
+                  flexibility in development, allowing for specific
+                  customizations without being bound to rigid system
+                  constraints.
+                </li>
+              </ul>
+            </div>
+          }
         />
         <Question
-          question="Can I pay for my subscription with an invoice?"
+          question="Can you assist with the actual deployment (terraform-like) in a cloud provider?"
           answer={
             <div>
-              Companies can use invoices to purchase Enterprise subscriptions.
-              Please{" "}
-              <MailTo
-                email="mailto:sales@amplication.com"
-                subject="Amplication Enterprise Plan Inquiry"
-                body="contact sales"
-              />{" "}
-              for more information and to purchase.
+              Amplication allows you to deploy to AWS's Elastic Container
+              Service (ECS). We provide in-depth documentation that will walk
+              you through using various Amplication plugins to provision the
+              required infrastructure and set up a process for continuous
+              integration and deployment.
+              <br />
+              <br />
+              The steps cover manual configurations for permissions and
+              authentication, plugin settings for AWS services, code generation,
+              provisioning with Terraform, and deployment using GitHub Actions.
             </div>
           }
         />
         <Question
-          question="We're an open-source project, is there special pricing available?"
+          question="I have an existing DB with a lot of data, entities, tables, and relationships. Can I use your backend development platform, while still keeping the data and the knowledge?"
+          answer="Amplication's DB Schema Import feature provides a clear pathway for integrating an existing database with the Amplication platform. The import feature is available on all Amplication plans, including Free accounts."
```

Comment:
The import is available only for the Enterprise plan...

Reply:
@PazYanoverr , right now the import is available on all plans actually. Do you plan to change this?
The rationale was that people would be able to import but if they are limited in number of entities that would block them. Of course now when we remove the limitation on #entities, we can reconsider. Anyway this is now part of the free plan as well.

Reply:
Yes, it appears in the spec- the import Prisma schema should be available in the Enterprise plan only.
The idea is to allow importing as part of the trial phase (if required by the user). After that- it won't be available in the free plan, as it's an advanced feature.